parser/sql/dialect/postgresql/src/main/java/org/apache/shardingsphere/sql/parser/postgresql/visitor/statement/type/PostgreSQLDDLStatementVisitor.java

```diff
@@ -419,6 +419,9 @@ public ASTNode visitAlterDefinitionClause(final AlterDefinitionClauseContext ctx
         if (null != ctx.alterTableActions()) {
             result.getValue().addAll(ctx.alterTableActions().alterTableAction().stream().flatMap(each -> getAlterDefinitionSegments(each).stream()).collect(Collectors.toList()));
         }
+        if (null != ctx.renameColumnSpecification()) {
```

Comment:
Please add sql parser test case for this change @ediconss , thank you.

Reply:
sorry,I don't know where to add，parser/sql/dialect/postgresql/src/test/java/org/apache/shardingsphere/test/it/sql/parser/it/postgresql/external/ExternalPostgreSQLParserIT.java already covered  relname col sql

Reply:
You can refer InternalSQLParserIT, there are lots of sql parse case now.

Reply:
https://github.com/apache/shardingsphere/blob/0a62ba2670f3eee6becf1e2aebaf1817c60282ee/test/it/parser/src/main/resources/sql/supported/ddl/alter-table.xml#L112

Reply:
So why this test case not find this bug? I think we should assert the new visit segment.

Reply:
https://github.com/apache/shardingsphere/blob/0a62ba2670f3eee6becf1e2aebaf1817c60282ee/test/it/parser/src/main/resources/case/ddl/alter-table.xml#L972-L974i , no assert